src/main/java/com/example/demo/controllers/SpeciesController.java

```diff
@@ -19,11 +21,44 @@
 public class SpeciesController {
 
     @Autowired
-    SpeciesRepository speciesRepository;
+    private SpeciesRepository speciesRepository;
+
+    @Autowired
+    private SpeciesService speciesService;
+
+
+    @GetMapping("/animals")
+    public List<Species> getAllAnimals(){
+        return speciesRepository.findSpeciesByIsAnimal(true);
+    }
+
+    @GetMapping("/animals/{season}")
+    public List<Species> getAnimalsBySeason(@PathVariable String season){
```

Comment:
Would be nice to filter using a request parameter, it'd make it more RESTful instead of having it as a "resource".